{"uid":"9ebb3860ae39dcfa","name":"Login with valid credentials test","fullName":"tests.test_login.TestLogin#test_valid_login","historyId":"e7f254b9bfd5f2bc8d4a1ffa6dc7aebb","time":{"start":1709514040298,"stop":1709514050570,"duration":10272},"description":"valid login","descriptionHtml":"<p>valid login</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5603ed0f4793 <unknown>\n#1 0x5603ecde81c6 <unknown>\n#2 0x5603ece33358 <unknown>\n#3 0x5603ece33411 <unknown>\n#4 0x5603ece76934 <unknown>\n#5 0x5603ece553fd <unknown>\n#6 0x5603ece73dd9 <unknown>\n#7 0x5603ece55173 <unknown>\n#8 0x5603ece262d3 <unknown>\n#9 0x5603ece26c9e <unknown>\n#10 0x5603ed0b88cb <unknown>\n#11 0x5603ed0bc745 <unknown>\n#12 0x5603ed0a52e1 <unknown>\n#13 0x5603ed0bd2d2 <unknown>\n#14 0x5603ed08917f <unknown>\n#15 0x5603ed0e2dc8 <unknown>\n#16 0x5603ed0e2fc3 <unknown>\n#17 0x5603ed0f3944 <unknown>\n#18 0x7f6356894ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7fbcbdf86010>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"574ab3fe69b013784529d6cc73f0d428\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fbcbdef4900>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5603ed0f4793 <unknown>\nE       #1 0x5603ecde81c6 <unknown>\nE       #2 0x5603ece33358 <unknown>\nE       #3 0x5603ece33411 <unknown>\nE       #4 0x5603ece76934 <unknown>\nE       #5 0x5603ece553fd <unknown>\nE       #6 0x5603ece73dd9 <unknown>\nE       #7 0x5603ece55173 <unknown>\nE       #8 0x5603ece262d3 <unknown>\nE       #9 0x5603ece26c9e <unknown>\nE       #10 0x5603ed0b88cb <unknown>\nE       #11 0x5603ed0bc745 <unknown>\nE       #12 0x5603ed0a52e1 <unknown>\nE       #13 0x5603ed0bd2d2 <unknown>\nE       #14 0x5603ed08917f <unknown>\nE       #15 0x5603ed0e2dc8 <unknown>\nE       #16 0x5603ed0e2fc3 <unknown>\nE       #17 0x5603ed0f3944 <unknown>\nE       #18 0x7f6356894ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_verify_url","time":{"start":1709514040296,"stop":1709514040296,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"base_url","time":{"start":1709514040296,"stop":1709514040296,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"secret_data","time":{"start":1709514040297,"stop":1709514040297,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"json_data","time":{"start":1709514040297,"stop":1709514040297,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"valid login","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5603ed0f4793 <unknown>\n#1 0x5603ecde81c6 <unknown>\n#2 0x5603ece33358 <unknown>\n#3 0x5603ece33411 <unknown>\n#4 0x5603ece76934 <unknown>\n#5 0x5603ece553fd <unknown>\n#6 0x5603ece73dd9 <unknown>\n#7 0x5603ece55173 <unknown>\n#8 0x5603ece262d3 <unknown>\n#9 0x5603ece26c9e <unknown>\n#10 0x5603ed0b88cb <unknown>\n#11 0x5603ed0bc745 <unknown>\n#12 0x5603ed0a52e1 <unknown>\n#13 0x5603ed0bd2d2 <unknown>\n#14 0x5603ed08917f <unknown>\n#15 0x5603ed0e2dc8 <unknown>\n#16 0x5603ed0e2fc3 <unknown>\n#17 0x5603ed0f3944 <unknown>\n#18 0x7f6356894ac3 <unknown>","statusTrace":"self = <tests.test_login.TestLogin object at 0x7fbcbdf86010>\nsecret_data = {'email': '', 'password': ''}\njson_data = {'forgot_password': {'error_message': \"We can't find a user with that e-mail address.\", 'success_message': 'A reset li...you page', 'name': 'test', 'new_name': 'another test', 'no_project_found_msg': 'No project matches the criteria', ...}}\n\n    @allure.description(\"valid login\")\n    @allure.title(\"Login with valid credentials test\")\n    @allure.tag(\"Tagged test\")\n    @pytest.mark.run(order=1)\n    def test_valid_login(self, secret_data: dict, json_data: dict):\n>       self.about_page.click_login_link()\n\ntests/test_login.py:39: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:38: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"574ab3fe69b013784529d6cc73f0d428\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7fbcbdef4900>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5603ed0f4793 <unknown>\nE       #1 0x5603ecde81c6 <unknown>\nE       #2 0x5603ece33358 <unknown>\nE       #3 0x5603ece33411 <unknown>\nE       #4 0x5603ece76934 <unknown>\nE       #5 0x5603ece553fd <unknown>\nE       #6 0x5603ece73dd9 <unknown>\nE       #7 0x5603ece55173 <unknown>\nE       #8 0x5603ece262d3 <unknown>\nE       #9 0x5603ece26c9e <unknown>\nE       #10 0x5603ed0b88cb <unknown>\nE       #11 0x5603ed0bc745 <unknown>\nE       #12 0x5603ed0a52e1 <unknown>\nE       #13 0x5603ed0bd2d2 <unknown>\nE       #14 0x5603ed08917f <unknown>\nE       #15 0x5603ed0e2dc8 <unknown>\nE       #16 0x5603ed0e2fc3 <unknown>\nE       #17 0x5603ed0f3944 <unknown>\nE       #18 0x7f6356894ac3 <unknown>\n\n.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Click Login link","time":{"start":1709514040298,"stop":1709514050570,"duration":10272},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5603ed0f4793 <unknown>\n#1 0x5603ecde81c6 <unknown>\n#2 0x5603ece33358 <unknown>\n#3 0x5603ece33411 <unknown>\n#4 0x5603ece76934 <unknown>\n#5 0x5603ece553fd <unknown>\n#6 0x5603ece73dd9 <unknown>\n#7 0x5603ece55173 <unknown>\n#8 0x5603ece262d3 <unknown>\n#9 0x5603ece26c9e <unknown>\n#10 0x5603ed0b88cb <unknown>\n#11 0x5603ed0bc745 <unknown>\n#12 0x5603ed0a52e1 <unknown>\n#13 0x5603ed0bd2d2 <unknown>\n#14 0x5603ed08917f <unknown>\n#15 0x5603ed0e2dc8 <unknown>\n#16 0x5603ed0e2fc3 <unknown>\n#17 0x5603ed0f3944 <unknown>\n#18 0x7f6356894ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/pages/base_page.py\", line 38, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/kisangateway-test/kisangateway-test/.venv/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"b49cc8863fc09a28","name":"public ip address","source":"b49cc8863fc09a28.txt","type":"text/plain","size":12},{"uid":"6b73b744bbe125c0","name":"Cookies","source":"6b73b744bbe125c0.json","type":"application/json","size":527},{"uid":"5513c1f13833ba11","name":"Session Storage","source":"5513c1f13833ba11.json","type":"application/json","size":2},{"uid":"f7b4f9f8824127af","name":"Local Storage","source":"f7b4f9f8824127af.json","type":"application/json","size":77},{"uid":"15fddca07e63d19","name":"Console Logs","source":"15fddca07e63d19.json","type":"application/json","size":310},{"uid":"bad186c551ac25b","name":"Full Page Screenshot","source":"bad186c551ac25b.png","type":"image/png","size":2477225},{"uid":"3812656e8dc7a238","name":"URL","source":"3812656e8dc7a238.uri","type":"text/uri-list","size":25},{"uid":"4887810761c1d5a9","name":"Network Logs","source":"4887810761c1d5a9.json","type":"application/json","size":1284988}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"tag","value":"Tagged test"},{"name":"feature","value":"Login"},{"name":"severity","value":"blocker"},{"name":"epic","value":"Security"},{"name":"tag","value":"run(order=1)"},{"name":"tag","value":"security"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"fv-az569-784"},{"name":"thread","value":"2207-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["security","Tagged test","run(order=1)"]},"source":"9ebb3860ae39dcfa.json","parameterValues":[]}